lightning/src/ln/channelmanager.rs

```diff
@@ -2009,17 +2009,6 @@ macro_rules! handle_new_monitor_update {
 					&$chan.context.channel_id());
 				Ok(false)
 			},
-			ChannelMonitorUpdateStatus::PermanentFailure => {
```

Comment:
This macro can just return a bool instead of a `Result` now, and rename `$update_res` to `$update_status`